.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: Build, test and deploy
+
+on: [push, pull_request, workflow_dispatch]
+
+env:
+  PYTHONWARNINGS: all
+
+jobs:
+  main:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2.2.1
+        with:
+          python-version: ${{ matrix.python }}
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('DEPENDS.txt') }}-${{ hashFiles('DEPENDS-tests.txt') }}-${{ hashFiles('DEPENDS-docs.txt') }}
+          restore-keys: ${{ runner.os }}-pip-${{ hashFiles('DEPENDS.txt') }}-
+
+      - name: Install dependencies
+        run: |
+          pip install -r DEPENDS.txt -r DEPENDS-tests.txt -r DEPENDS-docs.txt
+
+      - name: Build package
+        run: |
+          tools/header.py "Dependency versions"
+          tools/build_versions.py
+          python setup.py build_ext --inplace
+          pip install -e .
+
+      - name: Run tests
+        run: |
+          nosetests -v --exe --with-coverage --cover-package=skfuzzy skfuzzy
+          flake8 --exclude=test_* skfuzzy docs/examples
+      - name: Build docs
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          pip install "decorator>=4.3,<5"
```

Comment:
Temporary fix, we need to wait until NetworkX developers deal with decorator 5.x [Link to the issue](https://github.com/networkx/networkx/issues/4718)

Reply:
decorator 5.0.5 fixed this difficulty. So pip and conda and others should pick up the fixed decorator. NetworkX 2.5.1 also pins requirements to decorator <5 if you want that solution instead.

Reply:
Temporary fix removed. Thanks @dschult!